### DISCUSSION TITLE
reciper parser support for TudoGostoso

### DISCUSSION THREAD

**Discussion by paulo-erichsen** [mealie-recipes/mealie#1710] [Feature Request]
Hi, I'd like to ask if mealie could support parsing https://www.tudogostoso.com.br. That's a popular Brazilian recipe site and has lots of great recipes. 

Examples

- https://www.tudogostoso.com.br/receita/114-brigadeiro.html
- https://www.tudogostoso.com.br/receita/5211-massa-de-pastel.html
- https://www.tudogostoso.com.br/receita/2462-strogonoff-de-frango.html

**Comment by xarantolus** [comment]
The site [actually contains the required `ld+json` data](https://validator.schema.org/#url=https%3A%2F%2Fwww.tudogostoso.com.br%2Freceita%2F114-brigadeiro.html), but seems to block requests from what it deems to be bots. Not sure what would need to be done to circumvent this block

**Comment by michael-genson** [comment]
Recipe scrapers supports this, so it's likely related to this: https://github.com/mealie-recipes/mealie/issues/4024
